src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -182,6 +182,17 @@ private void copyContentSubFolders()
 
       private Section copySectionContent(Section section)
       {
+         var sectionWithRelativePath = new Section
+         {
+            Id = section.Id,
+            Reference = section.Reference,
+            Title = section.Title,
+            Sections = copySectionContents(section.Sections)
+         };
+
+         if (section.Content == null)

```

Comment:
@Yuri05 Indeed. When we change the contract...stuff breaks :). Should be good now